src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/how-to-upgrade-from-lts-2023-to-lts-2025/how-to-upgrade-audit-service.md

```diff
@@ -0,0 +1,255 @@
+
+```xml
+<extension target="org.nuxeo.ecm.platform.audit.service.NXAuditEventsService" point="event">
+    <event name="eventNameToAudit" />
+</extension>
+```
+
+Should be updated to:
+
+```xml
+<extension target="org.nuxeo.audit.service.AuditComponent" point="event">
+    <event name="eventNameToAudit" />
+</extension>
+```
+
+### Extended Info
+
+Extended Info extension point didn’t change, just the target of the contributions:
+
+```xml
+<extension target="org.nuxeo.ecm.platform.audit.service.NXAuditEventsService" point="extendedInfo">
+    <extendedInfo key="title" expression="${source.dublincore.title}" />
+</extension>
+```
+
+Should be updated to:
+
+```xml
+<extension target="org.nuxeo.audit.service.AuditComponent" point="extendedInfo">
+    <extendedInfo key="title" expression="${source.dublincore.title}" />
+</extension>
+```
+
+## Operation
+
+The `Audit.Query` operation has been removed from contributed operations because it was tied to SQL Audit Backend.
+
+If you were using it, we advise migrating your usage to the `Audit.QueryWithPageProvider`, which does the same thing
+while being compatible with all backend implementations.
+
+The `Audit.QueryWithPageProvider` now supports NXQL for the `query` parameter to query your Audit.
+
+## Nuxeo Server Configuration
+
+Nuxeo Server is now provided with an in memory implementation for Audit storage, which means that you need to install
+a Marketplace Package to store the audit in MongoDB, OpenSearch or a SQL database.
+
+### Audit in OpenSearch
+
+To store the Nuxeo audit in OpenSearch you need to install the [nuxeo-audit-opensearch1](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-audit-opensearch1)
+package.
+After installation, check that the `opensearch1-audit` template is included in your `nuxeo.conf` file. The template
+will enable the OpenSearch Audit Backend. The previous `audit.elasticsearch.enabled` configuration property has no more effect.
+
+Previously, the ElasticSearch / OpenSearch configuration property for Audit looked like:
+
+```properties
+# generic
+elasticsearch.addressList=http://localhost:9200
+elasticsearch.index.translog.durability=request
+elasticsearch.indexNumberOfShards=5
+elasticsearch.indexNumberOfReplicas=1
+elasticsearch.restClient.connectionTimeoutMs=30000
+elasticsearch.restClient.socketTimeoutMs=121000
+elasticsearch.restClient.ssl.certificate.verification=true
+
+# optional
+elasticsearch.restClient.username=
+elasticsearch.restClient.password=
+elasticsearch.restClient.truststore.path=
+elasticsearch.restClient.truststore.password=
+elasticsearch.restClient.truststore.type=
+elasticsearch.restClient.keystore.path=
+elasticsearch.restClient.keystore.password=
+elasticsearch.restClient.keystore.type=
+
+# audit specific
+audit.elasticsearch.indexName=${elasticsearch.indexName}-audit
+audit.elasticsearch.latestLogId.afterDate=now-14d/d
+```
+
+Now you will configure OpenSearch / ElasticSearch connection with:
```

Comment:
Mention ES 7 & 8? Or just remove Elasticsearch refs?

Reply:
yes maybe at the beginning something like:
The `nuxeo-audit-opensearch1` package is able to store Nuxeo audit in an OpenSearch 1.x, Elasticsearch 7.x or Elasticsearch 8.x cluster.